.gitignore

```diff
@@ -48,3 +48,5 @@ Thumbs.db
 *.mov
 *.wmv
 
+# TF version
+.terraform-version
```

Comment:
Is there a reason to add this in the gitignore file?

Reply:
I needed to run terraform fmt :P

Reply:
But is there a need to use a .terraform-version file here?